### PR TITLE
Hostapd bump

### PR DESCRIPTION
**Pull request by clefru**
Note: Unfortunately, I strongly disagree with the software development practices of hostapd and wpa_supplicant, which is: too long of a release cycle and no stable branch with backporting of fixes. Also I am not able to review or track upstream commits to see what's good for backporting. All I can do, is send a PR and hope that upstream testing was sufficient. Given that 2.7 ships 2 years worth of changes, I do expect a fallout however! :(

###### Motivation for this change
Upstream release

###### Things done
Pushed to my laptop to test wpa_supplicant.
Pushed to my nixos access point to test hostapd.

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by vcunat** [comment]
Some small fallout risk seems normal for master/unstable.  The wording made me think you filed this against the stable 18.09 branch :-)

**Review by dasJ — APPROVED** [review]
Looks good, you could add a comment to the patch mentioning that it's not a security patch but a build fix

**Comment by dasJ** [comment]
Sorry, it was on the other side of the huge removal block in the diff. Never mind about my previous statement

**Comment by delroth** [comment]
I had #55926 pending to bump wpa_supplicant but no reviews in a week. Feel free to close that PR if this one gets merged.

**Comment by ryantm** [comment]
@grahamcofborg build hostapd wpa_supplicant

**Comment by ryantm** [comment]
I'm kind of surprised so little depends on wpa_supplicant.
@vcunat Are you okay with merging this? It looks okay to me, but I didn't test it.

**Comment by clefru** [comment]
> I had #55926 pending to bump wpa_supplicant but no reviews in a week. Feel free to close that PR if this one gets merged.

Sorry for missing your PR. I just used hostapd as a search term.

**Comment by ryantm** [comment]
@xeji merged the other one, so this needs to be thinned out to be just `hostapd`, @clefru

**Comment by vcunat** [comment]
I haven't done any kind of testing either, not even looking at NEWS.  I can't see anything blocking this either, but it would be better if someone actually used it before merging :-)

**Comment by clefru** [comment]
I removed the wpa_supplicant part from this PR as it was pushed in the other.

**Comment by clefru** [comment]
@vcunat hostapd 2.7 is running on my AP since two or three days. Haven't noticed anything bad.

**Comment by ryantm** [comment]
@GrahamcOfBorg build hostapd

**Comment by ryantm** [comment]
@clefru thanks; please consider adding yourself as a maintainer.

**Pull request merged**
